python/cudf/cudf/tests/test_stats.py

```diff
@@ -26,17 +27,18 @@ def test_series_reductions(method, dtype):
         mask = arr > 0.5
 
     arr = arr.astype(dtype)
-    arr2 = arr[mask]
     sr = Series.from_masked_array(arr, Series(mask).as_mask())
```

Comment:
This only creates nulls, what happens if we have `NaN` values as well?

Reply:
So this was the problem..
 Lets say `mask` has 12 `False` values, `arr2` will now have 88 elements only as numpy will filter out elements from `arr` where `mask` is set to `False`. So `sr` will have say 100 elements, with 12 `Null` values.

Due to mismatch between lengths of `arr2` & `sr`, operations like `mean` will fail. Plus, since we are adding `skipna` param to the test, it will fail. Thus creating a Pandas series and assigning null values at `mask`.

Reply:
I guess my point is we should test for a combination of `NaN` and `null` values in cuDF to ensure things are working as expected. For the equivalent Pandas tests we can just use `NaN`s.

Reply:
Got it, added both `nan` and `null` to cudf and pandas will now have `nan`